crates/agglayer-contracts/build.rs

```diff
@@ -1,26 +1,32 @@
 use std::path::Path;
 
 fn main() {
-    let contract_dir = Path::new(env!("CARGO_MANIFEST_DIR")).join("src");
+    let contract_dir = Path::new(env!("CARGO_MANIFEST_DIR")).join("src/contracts");
 
     build_contract(
```

Comment:
Note: `ethers` is deprecated. We should switch to alloy contract build in the foreseeable future.